eZ/Publish/Core/Persistence/Legacy/Content/FieldValue/Converter/DateConverter.php

```diff
@@ -93,10 +92,8 @@ public function toFieldDefinition(StorageFieldDefinition $storageDef, FieldDefin
         // Building default value
         switch ($fieldDef->fieldTypeConstraints->fieldSettings['defaultType']) {
             case DateType::DEFAULT_CURRENT_DATE:
-                $dateTime = new DateTime();
-                $dateTime->setTime(0, 0, 0);
                 $data = array(
-                    'timestamp' => $dateTime->getTimestamp(),
+                    'timestamp' => time(),
                     'rfc850' => null,
```

Comment:
Like DateAndTime we should perhaps add `'timestring' => 'now',` here. It was added by @glye I think, to make sure cached values would not cause issues.

Reply:
Yes, see https://github.com/ezsystems/ezpublish-kernel/pull/1870 and follow-up https://github.com/ezsystems/ezpublish-kernel/pull/1916. The point is to not use timestamps, but rather DateIntervals. An interval can safely be cached. The actual date/time is calculated when you instantiate it.